config/data_sync_list/common.json

```diff
@@ -1,4 +1,20 @@
 {
+    "Files": [
+        {
+            "Path": "/var/log/obmc-console.log",
+            "Description": "Host console persisted log data",
+            "SyncDirection": "Active2Passive",
+            "SyncType": "Periodic",
```

Comment:
Actually, it doesn't need to be periodic since the update only happens during IPL, and such changes are rare in the field. Once we have deferred sync support (or the state-driven sync), it could be updated accordingly. But for now, this approach should be fine.